CalibFormats/CaloObjects/src/CaloSamples.cc

```diff
@@ -3,14 +3,13 @@
 #include <math.h>
 #include <iostream>
 
-const int CaloSamples::MAXSAMPLES;
-
```

Comment:
Did you check that it is really the same `MAXSAMPLES` that is used elsewhere? From a quick check it looks like it is also not used somewhere else.
Sorry to be so picky, but I don't like to have this only half-way removed.